doc/index.rst

```diff
@@ -34,9 +34,24 @@ There are three different ways which you can use Cube Browser:
 Installation
 ____________
 
-Conda is great
+In the following pages you will find downloads of Jupyter notebooks containing Cube Browser operations.
+To be able to use these notebooks, you will need to configure your environment correctly.
+The most efficient and clean method to do this is to use a Conda environment (please see `Anaconda website <https://anaconda.org/>`_ for help and information).
+
+If you have never used Conda before, you will need to follow these `quick installation instructions <http://conda.pydata.org/docs/install/quick.html>`_ to get started.
+
+Once you have done this, open a terminal and type:
+
+``conda create -n cb -c scitools``
+
+This will create your environment to the specific requirements necessary to run Cube Browser.
+To activate that environment, type:
+
+``. activate cb``
```

Comment:
This won't work on Windows. Also, why is everyone so against `source activate ...`? That's even what conda tells you to do!